src/vite-env.d.ts

```diff
@@ -1,11 +1,14 @@
 // / <reference types="vite/client" />
+// To ensure it is treated as a module, add at least one `export` statement
+export {};
```

Comment:
Can we omit it after removing module `import` statement?

Reply:
Without this {} , we get router types error, doesn't it?

```
src/application/router/index.ts:1:10 - error TS2305: Module '"vue-router"' has no exported member 'createRouter'.

1 import { createRouter, createWebHistory } from 'vue-router';
           ~~~~~~~~~~~~

src/application/router/index.ts:1:24 - error TS2305: Module '"vue-router"' has no exported member 'createWebHistory'.

1 import { createRouter, createWebHistory } from 'vue-router';
```

Reply:
yes, you are right)